src/CEOFESABundle/Entity/Parcours.php

```diff
@@ -253,13 +258,15 @@ public function setPrcPresence($prcPresence)
     /**
      * @return int
      */
-    public function getTotalHeures()
+    public function getTotalHeures(\DateTime $date)
```

Comment:
Yes, I think. I have check CeofesaBundle Repository with getTotalHeures, and Ressources repository with .totalHeures